tests/posix/common/prj.conf

```diff
@@ -5,4 +5,3 @@ CONFIG_ZTEST=y
 CONFIG_SEM_VALUE_MAX=32767
 CONFIG_POSIX_MQUEUE=y
 CONFIG_HEAP_MEM_POOL_SIZE=4096
-CONFIG_TEST_EXTRA_STACKSIZE=256
```

Comment:
Hmm. Wonder why this was hard-coded in the test case instead of just increasing the stack size in the test itself. Good to remove here.